### ISSUE TITLE
[Search] Turning on and off filter gives no results

### ISSUE THREAD

**Issue by shobart** [nasa/openmct#720]
Replication steps (from a freshly started WARP):
1. In the search field, enter "temp".  Observe lots of results.
2. Filter by "Folder".  Observe filtered results.
3. Uncheck the filter box, and observe that under the search box it say "Filtered by:  NONE"
4. In the search field, enter "temp".  Observe it brings back no results.  

Workaround:  

After step 4, click on the "clear" button (x within a circle) next to the "Filtered by:  NONE"
Now the search works again.

**Comment by charlesh88** [comment]
Did this get fixed with #504? Adding unverified to see if this can be replicated in a Testathon.

**Comment by akhenry** [comment]
Think this got missed in the testathon as it was open. It's definitely a partial dupe of #1116, which is addressed in its own ticket #1117 and fixed in #1135. Closing, and setting milestone to Rajaniemi for verification.

**Comment by VWoeltjen** [comment]
Checkboxes are behaving rationally now, but after step 4 I see No Results, instead of a return of the many results I saw after step 1

**Comment by VWoeltjen** [comment]
Updated title to reflect way the issue currently manifests

Since I'm seeing results after step 1, don't think this behavior is related to #1279

**Comment by charlesh88** [comment]
Still a problem here, but different from original description. Seen running locally with local storage:

1. In the search field, enter a string that gives results for a variety of object types, one of more should be of type folder. Observe results.
1. Filter by "Folder". Observe filtered results: they should be less than the previous step.
1. Uncheck the filter by "Folder" checkbox, and observe that no results are found.
1. In the search field, re-enter the same string as step 1.  Observe that results are now displayed as in step 1.
